controllers/tasks.js

```diff
@@ -13,8 +13,16 @@ exports.getTasks = async (req, res, next) => {
 
 // @desc    Get single task
 // @route   GET /api/v1/tasks/:id
-exports.getTask = (req, res, next) => {
-  res.status(200).json({ success: true, msg: `Show task ${req.params.id}` });
+exports.getTask = async (req, res, next) => {
+  try {
+    const task = await Task.findById(req.params.id);
+    if (!task) {
+      return res.status(400).json({ success: false });
+    }

```

Comment:
In case the task wasn't found you can use `404`. You can apply it not only to routes or pages.

Reply:
Thanks.